streamlit_app/app.py

```diff
@@ -136,13 +136,12 @@ def get_models(llm_models, vision_models):
 
     tools = st.multiselect(
         "Select tools to include:",
-        # ["Search", "Location", "Vision", "Stocks", "News"], #all options available
-        # ["Search", "Location", "Vision", "Stocks", "News"],) #options that are selected by default
-        ["Search", "Location", "Vision", "Stocks"],  # all options available
-        ["Search", "Location", "Vision", "Stocks"],
+        ["search", "locate", "vision", "stocks"],  # all options available
+        ["search", "locate", "vision", "stocks"],
```

Comment:
Could you move `# options that are selected by default` annotation next to the second list?